scripts/test.mk

```diff
@@ -0,0 +1,52 @@
+# This file is part of the repository template project. Please, DO NOT edit this file.
```

Comment:
An alternative would be to move these `test-*` make targets into the `./Makefile`, which could further emphasise the importance of test automation. Nevertheless, as I understand it, the intent here is to establish a well-defined set of test types and to set up the appropriate hooks within the `./scripts/test/...` directory. There is no expectation this file changes much. This looks like a good pattern.

Reply:
I'm all up for whatever changes you think might make sense, but I do worry a little about that root `Makefile` getting a bit cluttered.  I do like the `init.mk` separation that was already there.

And yes, that is absolutely the intent: make it easy to do the right thing, to figure out what the right thing is, and to implement it in a way that is coherent between projects.